crates/client/rpc/src/types.rs

```diff
@@ -25,6 +29,14 @@ pub enum ParseTokenError {
     ParseFailed(ParseIntError),
 }
 
+#[derive(Serialize, Deserialize, Debug, Eq, PartialEq)]
+pub struct CustomDeclareV0Transaction {
+    pub declare_transaction: DeclareTransactionV0V1,
+    pub program_vec: Vec<u8>,
+    pub entrypoints: IndexMap<EntryPointType, Vec<EntryPoint>>,
+    pub abi_length: usize,
+}
+
```

Comment:
Don't you already declare this type in `crates/client/rpc-core/src/lib.rs`

Reply:
I have this type in `crates/client/rpc-core/src/lib.rs` but it can't be imported into `crates/client/rpc/src/lib.rs` so had to define it explicitly here only.

Reply:
There is no reason it cannot be imported from rpc-core to rpc. 
Rpc depends on rpc-core

Reply:
wait sorry It was other way around. fixed this in new commit.